### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by henrikef**
This branch is to implement the new logging features of threeML. Currently untested & not ready to merge.

### CODE REVIEW

**Comment by henrikef** [comment]
With the logging features now implemented in threeML, this should be ready to merge from my side. Review would be appreciated.

**Review by omodei — APPROVED** [review]
Everything seems ok, in my opinion we can merge this.

File: hawc_hal/HAL.py [hunk 1]
```diff
@@ -265,34 +267,34 @@ def display(self, verbose=False):
         Prints summary of the current object content.
         """
 
-        print("Region of Interest: ")
-        print("-------------------\n")
+        log.info("Region of Interest: ")
+        log.info("-------------------\n")
         self._roi.display()
```
Comment (grburgess) [inline hawc_hal/HAL.py:272]: Only a style call :) but do you want the return chars there?

**Review by grburgess — APPROVED** [review]

File: hawc_hal/HAL.py (hunk 1 above)
Comment (henrikef) [reply hawc_hal/HAL.py:272]: I don't really care, I just copied it from the original 😆 I can remove the \n.

**Comment by grburgess** [comment]
Looks cool @henrikef !

**Pull request merged**